src/components/tutorials-sidebar/index.tsx

```diff
@@ -49,32 +52,78 @@ function CollectionViewSidebarContent({
 					<Fragment key={title}>
 						<HorizontalRule />
 						{title ? <SectionTitle text={title} /> : null}
-						<SectionList items={items} />
+						<SectionList>
+							{items.map(({ text, href, isActive }: ListItemProps) => {
+								return (
+									<ListItem
+										key={`${text}${href}`}
+										text={text}
+										href={href}
+										isActive={isActive}
+									/>
+								)
+							})}
+						</SectionList>
 					</Fragment>
 				)
 			})}
 		</>
 	)
 }
 
-function TutorialViewSidebarContent({ items }: SectionListProps) {
-	return <SectionList items={items} />
+function TutorialViewSidebarContent({
+	items,
+}: {
+	items: TutorialListItemProps[]
+}) {
+	return (
+		<SectionList>
+			{items.map(
+				({
+					text,
+					href,
+					isActive,
+					tutorialId,
+					collectionId,
+				}: TutorialListItemProps) => {
+					return (
+						<TutorialListItem
+							key={`${collectionId}${tutorialId}`}
+							text={text}
+							href={href}
+							isActive={isActive}
+							tutorialId={tutorialId}
+							collectionId={collectionId}
+						/>
+					)
+				}
+			)}
+		</SectionList>
+	)
+}
+
+function SectionList({ children }: SectionListProps) {
+	return <ul className={s.listRoot}>{children}</ul>
 }
 
-function SectionList({ items }: SectionListProps) {
+function TutorialListItem({
+	href,
+	isActive,
+	text,
+	tutorialId,
+	collectionId,
+}: TutorialListItemProps) {
+	/**
+	 * Query for progress, and display the appropriate status icon
+	 */
+	const { tutorialProgressStatus } = useTutorialProgress({
+		tutorialId,
+		collectionId,
+	})
+	const rightIcon = <TutorialProgressIcon status={tutorialProgressStatus} />
```

Comment:
💭 perhaps we could make a wrapper component here that is like `TutorialListItemWithProgress`, in case we want to render this component elsewhere and not show the progress data.

Reply:
See an example with the tutorial card here https://github.com/hashicorp/dev-portal/blob/main/src/components/tutorial-card/helpers/with-bookmark.tsx

Reply:
If we go with the collection call route, then disregard my comments above.

Reply:
If we want to render this component without progress data, I think we can use `SidebarNavMenuItem` directly. 

That being said, can definitely see why we might want something like `with-bookmark` or even `connected-bookmark-button` for progress... Had noodled on that a bit in the spike with a higher-order-component, see [connect-tutorial-progress-component](https://github.com/hashicorp/dev-portal/blob/7785c299ab24658922ce28df0e6dce2efc78ee8a/src/components/tutorial-card/components/connect-tutorial-progress-component/index.tsx).

Would it be alright if we defer the refinement here to a future PR? I feel like splitting out something like `connect-tutorial-progress-component` might come more naturally in adding the connected tutorial card.

Let me know what you think 🙇‍♂️